dashboard/config/application.rb

```diff
@@ -128,5 +128,10 @@ class Application < Rails::Application
 
     # turn off ActionMailer logging to avoid logging email addresses
     ActionMailer::Base.logger = nil
+
+    if Rails.env.production?
+      require 'newrelic_rpm'
```

Comment:
My hunch is that this line is a no-op, as the above `Bundler.require(:default, Rails.env)` (on line [20](https://github.com/code-dot-org/code-dot-org/pull/8354/files#diff-5fdb0cec2f0205f0e50d1df7631fb1e4R20)) will already automatically `require` all gems not marked `require: false` referenced by our `Gemfile` for the specified groups.

However, it's not doing any harm to explicitly include it here, so should be fine to leave as-is (you'd need to do another full round of testing if it's removed).